### ISSUE TITLE
I can't find any raytrace examples

### ISSUE THREAD

**Issue by trsh** [but0n/Ashes#43]
Any pointers?

**Comment by but0n** [comment]
Here is the ray tracer demo:
https://github.com/but0n/Ashes/blob/master/bvh.html

**Comment by trsh** [comment]
Well it's not working. I got light brown screen and "1" printed in console

**Comment by but0n** [comment]
> Well it's not working. I got light brown screen and "1" printed in console

Is there other error message in console?
Can you provide the screenshot?

Currently some assets such as HDR environment textures might not available in this repository, perhaps this caused an 404 error, and the it interrupted.

**Comment by trsh** [comment]
No other messages in console. And yes One texture was missing, I replaced it with something, but still nothing...

**Comment by but0n** [comment]
OK, I will clean the code, make sure all these textures are available.

**Comment by but0n** [comment]
Those textures has been removed.
![image](https://user-images.githubusercontent.com/7625588/76134329-75136e00-6071-11ea-868f-db596d4f384a.png)

Press `K` to toggle debug mode and rendering mode.
Press `U` & `J` for camera up and down.

**Comment by trsh** [comment]
So I fallowed the instructions
```
git clone https://github.com/but0n/Ashes.git
cd Ashes

# if you don't have yarn installed
npm install -g yarn

yarn

yarn dev
```

opened `http://127.0.0.1:8080/bvh.html` in firefox&chrome (Webgl2 enabled) and nothing happens.. just slightly brown background, "1" in console and after some while the Tab starts to Hand the whole browser.

**Comment by but0n** [comment]
You can try `git clone --recursive https://github.com/but0n/Ashes.git` if `Ashes/gltfsamples` is empty.

**Comment by trsh** [comment]
Just did. Same thing

**Comment by trsh** [comment]
![2020-03-07_151728](https://user-images.githubusercontent.com/3727523/76144232-d4c84400-6086-11ea-8a6b-b71b0317bfa6.png)

**Comment by trsh** [comment]
![2020-03-07_151728](https://user-images.githubusercontent.com/3727523/76144242-fcb7a780-6086-11ea-8e6a-4b62830b21b2.png)

**Comment by but0n** [comment]
Hmmm, it's weird, seems it stopped after download the first hdr cube texture: https://github.com/but0n/Ashes/blob/master/bvh.html#L94

It should be like this:

![image](https://user-images.githubusercontent.com/7625588/76154561-94be9b00-6132-11ea-8ac2-d6ee19461e99.png)

**Comment by trsh** [comment]
I dont know how to fix this.

**Comment by but0n** [comment]
I tried again with following command and it works fine: 
```
git clone --recursive https://github.com/but0n/Ashes.git a2 && cd a2 && yarn && yarn dev
```
Can you visit this website and send me a report?
https://webglreport.com/?v=2

**Comment by but0n** [comment]
Thank you, can you go to the `gltfsamples` folder see is there has `DamagedHelmet.glb`?

**Comment by trsh** [comment]
Yes its there

**Comment by but0n** [comment]
Thank you, I will find a similar environment and try to figure out what's going on.

**Comment by Muini** [comment]
Hi ! I just tried and got the same problem.
However, I noticed missing files :
![image](https://user-images.githubusercontent.com/6160740/82248299-e2673900-9915-11ea-9926-6ecc8c36316d.png)
I checked and there is no .hdr in envmap folder, not on master branch anyway, maybe you need git LFS to commit them. I would love to try those demo too :)

**Comment by but0n** [comment]
@Muini 
Yeah, I haven't upload HDR textures yet due to certain reasons, can you check out `568bb22` ? That version doesn't requires any HDR textures. Thank you!
